pkg/reconciler/testing/storage.go

```diff
@@ -54,6 +54,12 @@ func WithStorageBucket(bucket string) StorageOption {
 	}
 }
 
+func WithStorageProject(project string) StorageOption {
+	return func(s *v1alpha1.Storage) {
+		s.Spec.Project = project
+	}
+}
+
 func WithStorageEventTypes(eventTypes []string) StorageOption {
```

Comment:
Is this used at all?

Reply:
yes, this does not refer to the `EventType` CRD. Is the list of string event types you can configure in the Storage source